docs/repositories-configure/codacy-configuration-file.md

```diff
@@ -122,7 +120,6 @@ phpmd
 pmd
 prospector
 psscriptanalyzer
-pylint
 pylintpython3
 remark-lint
 revive
```

Comment:
@codacy-vrhpires The [list you provided](https://codacy.atlassian.net/browse/DOCS-619) is missing `roslyn`, which is a [supported tool for Unity](https://docs.codacy.com/getting-started/supported-languages-and-tools/). Can you please validate? 🙏

Reply:
It's a client-side tool and the list doesn't include any of them.

Reply:
Right! So it shouldn't be on this list. I removed it. 👍